# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1233109178 18000
#      Tue Jan 27 21:19:38 2009 -0500
# Node ID 0611b48a2b61e6fa823610868fa93fc34bba9276
# Parent  9fc72c114ce177c7d29a275efb1e1f8b8e9e6074
file-io.cc (Ffprintf): doc fix

diff --git a/src/file-io.cc b/src/file-io.cc
--- a/src/file-io.cc
+++ b/src/file-io.cc
@@ -809,16 +809,17 @@ from the beginning of the file @var{fid}
   return retval;
 }
 
 DEFUN (fprintf, args, nargout,
   "-*- texinfo -*-\n\
 @deftypefn {Built-in Function} {} fprintf (@var{fid}, @var{template}, @dots{})\n\
 This function is just like @code{printf}, except that the output is\n\
 written to the stream @var{fid} instead of @code{stdout}.\n\
+If @var{fid} is omitted, the output is written to @code{stdout}.\n\
 @seealso{printf, sprintf, fread, fscanf, fopen, fclose}\n\
 @end deftypefn")
 {
   static std::string who = "fprintf";
 
   octave_value retval;
 
   int result = -1;
